CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Features ✨ and improvements 🏁
+* Added new example `SecondaryDisplayPresentationActivity` to showcase how to use [Presentation](https://developer.android.com/reference/android/app/Presentation) to show a `Mapview` on a secondary display. ([1905](https://github.com/mapbox/mapbox-maps-android/pull/1905))
```

Comment:
sadly example are not part of the SDK so they do not deserve changelog entry...

Reply:
Why shouldn't we just add this functionality to the SDK though?

Reply:
😢

Reply:
@yunikkk do you mean that we should have our own Presentation that adds the right lifecycle events and expose that?

Reply:
The issue I see here is that the `Presentation` is a class provided by Android itself.

Moreover, we have no control at all on the `Activity` that it is used to host that `Presentation` dialog and it doesn’t inherit from the `AppCompatActivity` one. It uses the standard/vanilla `Activity`.

I honestly don’t think that we should maintain a custom `Presentation`, I would rather see the users do it if they really need it ’cause they might need other customization in their `Presentation` implementation.